subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/git/tasks/GitCommitImpl.java

```diff
@@ -35,13 +35,16 @@ static String getAggregatedCommitMessage(List<String> descriptions) {
         return result.toString();
     }
 
-    static List<String> getAddCommand(List<File> files) {
+    static List<String> getAddCommand(List<File> files, List<File> directories) {
         List<String> args = new ArrayList<>();
         args.add("git");
         args.add("add");
         for (File file : files) {
             args.add(file.getAbsolutePath());
         }
+        for (File directory : directories) {
+            args.add(directory.getAbsolutePath());
```

Comment:
Why do we need files and directories given they are handled in exact the same way? Can you just use files?